src/main/scala/CypherExpressable.scala

```diff
@@ -25,4 +26,16 @@ object CypherExpressible {
     def toQuery(double: Double): String =
       double.toString
   }
+
+  implicit object CypherExpressibleBoolean extends CypherExpressible[Boolean] {
+    def toQuery(boolean: Boolean): String =
+      if (boolean) "TRUE" else "FALSE"
+  }
+
+  implicit def optionExpressible[T](implicit serializer: CypherExpressible[T]): CypherExpressible[Option[T]] =
```

Comment:
what did you reference for this? i *think* it gets iffy if you have nested containers, because it won't figure out how to apply the implicit conversions. might want to check. `Option[Option[T]]` would be a compilation error if i'm right :D